src/proj7BittingCerratoCohenEllmer/bantam/lexer/Scanner.java

```diff
@@ -288,7 +325,7 @@ private Token createToken(String spelling) {
                     sourceFile.getCurrentLineNumber());
         }
 
-        if (isValidInt(spelling)) {
+        if (isInt(spelling)) {
```

Comment:
All of these is... methods duplicate a lot of our code. I know that we'll need some of them, but this should be a target for later elegance improvements.